config/routes.rb

```diff
@@ -168,30 +165,30 @@
 
   resources :stories, only: [:show]
 
-  get '/leaderboards' => 'leaderboards#index'
+  get '/leaderboards', to: 'leaderboards#index'
 
   # Webhooks
   namespace :webhooks do
-    post '/assembly_assets/transaction' => 'assembly_assets#transaction'
-    post '/mailgun' => 'mailgun#create'
-    post '/mailgun/reply' => 'mailgun#reply'
-    post '/github' => 'github#create'
-    post '/landline' => 'landline#create'
-    post '/readraptor/immediate/:entity_id' => 'read_raptor#immediate', as: :readraptor_immediate
-    post '/readraptor/daily'          => 'read_raptor#daily'
-    post '/readraptor/unread_comment' => 'read_raptor#unread_coment', as: :readraptor_unread_comment
-    post '/pusher' => 'pusher#auth'
+    post '/assembly_assets/transaction', to: 'assembly_assets#transaction'
+    post '/mailgun', to: 'mailgun#create'
+    post '/mailgun/reply', to: 'mailgun#reply'
+    post '/github', to: 'github#create'
+    post '/landline', to: 'landline#create'
+    post '/readraptor/immediate/:entity_id', to: 'read_raptor#immediate', as: :readraptor_immediate
+    post '/readraptor/daily'         , to: 'read_raptor#daily'
+    post '/readraptor/unread_comment', to: 'read_raptor#unread_coment', as: :readraptor_unread_comment
```

Comment:
Line is too long. [102/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.